### DISCUSSION TITLE
Infinite canvas

### DISCUSSION THREAD

**Discussion by JurajJakubov** [CommunityToolkit/Maui#561] [New Feature Discussions]
I am wondering if is possible to create infinite canvas. So user can move canvas as much as he want to left, right, top, bottom... Is it possible?

**Comment by VladislavAntonyuk** [comment]
You can put DrawingView inside the ScrollView and dynamically change the size when user scrolls.

you can also simulate such action by updating all points locations.

and the final thought. You can play with ICanvas which is accessible from DrawingView Action. It should allow applying changes to the whole canvas.

**Reply by JurajJakubov** [reply]
> You can put DrawingView inside the ScrollView and dynamically change the size when user scrolls.
> 
> you can also simulate such action by updating all points locations.
> 
> and the final thought. You can play with ICanvas which is accessible from DrawingView Action. It should allow applying changes to the whole canvas.

Great ideas. Thank you very much for your input I never thought in these ways. Anyway it will be still awesome to have this native like here https://docs.microsoft.com/en-us/windows/communitytoolkit/controls/infinitecanvas

**Reply by VladislavAntonyuk** [reply]
Contributions are welcome 😄

**Reply by JurajJakubov** [reply]
Yeah I am sure :D hopefully once I learn c#.

**Comment by CliffAgius** [comment]
Closed as answered
